lg_stats/msg/Stats.msg

```diff
@@ -0,0 +1,4 @@
+string lg_system_name
```

Comment:
This needs to be more elastic multi-purpose type of message e.g. like [here](https://github.com/EndPointCorp/portal-ros/blob/development/catkin/src/statistics/msg/StatsD.msg).  Instead of system name it should prolly be a tree like: systemname.application.event (for events) or systemname.application.event_rate (for timeseries data)